PSSwagger/New-ServiceClient.ps1

```diff
@@ -0,0 +1,26 @@
+Microsoft.PowerShell.Core\Set-StrictMode -Version Latest
```

Comment:
Any reason why you don't want to add to GeneratedHelpers?

Reply:
Module owner can just replace this file during the mock testing.